IntegrationTests/common/script/Makefile.mk

```diff
@@ -70,7 +76,7 @@ $(DEPS)/%.d: $(EMDATA)/MemPrintsCM
 	@set -e; rm -f $@; mkdir -p $(DEPS); \
 	 TOP_FUNC=`echo $@ | sed 's,.*\/\([^/]*\)\.d,\1,g'`; \
 	 TOP_FILE=`grep -l $${TOP_FUNC} $(TOP_FUNCS)/*.cc`; \
-	 $(XILINX_GCC) -MM -I$(TRK_ALGO) -I$(TOP_FUNCS) -I$(XILINX_VIVADO)/include $${TOP_FILE} > $@.$$$$; \
+	 $(XILINX_GCC) -MM -I$(TRK_ALGO) -I$(TRK_ALGO)/Project -I$(TOP_FUNCS) -I$(XILINX_VIVADO)/include $${TOP_FILE} > $@.$$$$; \
```

Comment:
@aryd I think it existed at some point, but there is no TrackletAlgorithm/Project directory on this branch now.

Reply:
Oops, not sure what happened to this directory - I had it locally. I have now pushed this (again).